src/server.ts

```diff
@@ -0,0 +1,58 @@
+import * as express from 'express'
+import * as ctx from './ApplicationContext'
+import * as session from 'express-session'
+import * as sessionFileStore from 'session-file-store'
+import * as cookieParser from 'cookie-parser'
+import {Auth} from './identity/auth'
+import * as log4js from 'log4js'
+import * as config from './config'
+
+const logger = log4js.getLogger('server')
+const expressNunjucks = require('express-nunjucks')
+
+const appRoot = require('app-root-path')
+
+const {PORT = 3005} = process.env
+const app = express()
+const FileStore = sessionFileStore(session)
+
+log4js.configure(config.LOGGING)
+
+expressNunjucks(app, {})
```

Comment:
We shouldn't be changing the templating engine unless we're planning on migrating the UI as well. And if that's the case, good bye typescript.

Reply:
We had a discussion with Yuksel and Joel about the template engine. Svelte doesn't work as an app engine in Express wich makes unit testing difficult. Joel was keen to use Nunjucks as it is used extensivley with GDS design/development. We should maybe have this discussion again.